rules/windows/command_and_control_new_terms_commonly_abused_rat_execution.toml

```
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2023/04/03"
+integration = ["windows", "system", "endpoint"]
+maturity = "production"
+min_stack_comments = "The New Term rule type used in this rule was added in Elastic 8.4"
+min_stack_version = "8.4.0"
+updated_date = "2023/04/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may install legitimate remote access tools (RAT) to compromised endpoints for further command-and-control
+(C2). Adversaries can rely on installed RATs for persistence, execution of native commands and more. This rule detects
+when a process is started whose name or code signature resembles commonly abused RATs. This is a New Terms rule type
+indicating the host has not seen this RAT process started before within the last 30 days.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-system.*", "logs-windows.*"]
```

```suggestion
index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
```
Removing the system one as the default windows process logs don't contain the signature or original file name information